src/dsl.rs

```diff
@@ -356,6 +356,15 @@ pub struct StepTypeWGHandler<F, Args, D: Fn(&mut StepInstance<F>, Args) + 'stati
 }
 
 impl<F, Args, D: Fn(&mut StepInstance<F>, Args) + 'static> StepTypeWGHandler<F, Args, D> {
+    pub fn new(id: UUID, annotation: &'static str, wg: D) -> Self {
```

Comment:
How was this struct created before?

Reply:
with the `step_type_def()` function, but for testing adding a `new` method made it so that a new circuit wouldn't need to be created. Let me know if this shouldn't be here.